server/routes/appointments/feedback/shared/sessionFeedback/sessionFeedbackView.ts

```diff
@@ -146,6 +146,7 @@ export default class SessionFeedbackView {
         },
         {
           divider: 'or',
+          class: 'govuk-label--m',
```

Comment:
Is this related to this change or you are fixing something that you found wrong?

Reply:
I think the default divider text was too small and didn't match the design.